oauth.go

```diff
@@ -438,11 +438,11 @@ func encodeUserParams(userParams map[string]string) string {
 
 // DEPRECATED: Use Post() instead.
 func (c *Consumer) PostForm(url string, userParams map[string]string, token *AccessToken) (resp *http.Response, err error) {
-	return c.Post(url, userParams, token)
+	return c.Post(url, "", userParams, token)
 }
 
-func (c *Consumer) Post(url string, userParams map[string]string, token *AccessToken) (resp *http.Response, err error) {
-	return c.makeAuthorizedRequest("POST", url, LOC_BODY, "", userParams, token)
+func (c *Consumer) Post(url string, body string, userParams map[string]string, token *AccessToken) (resp *http.Response, err error) {
```

Comment:
Thanks for the suggestion Matt, that make sense. I'll create a new method for that.